models/daily_spend.sql

```diff
@@ -82,7 +82,8 @@ storage_spend_daily as (
             ),
             0
         ) as spend,
-        spend as spend_net_cloud_services
+        spend as spend_net_cloud_services,
+        any_value(daily_rates.currency) as currency
```

Comment:
```suggestion
        any_value(coalesce(daily_rates.currency, latest_rates.currency)) as currency
```

Can we apply this change here and to the other CTEs? When daily rates are missing, we fall back to the most recent rate. Same should apply to currency.